ckanext/canada/templates/internal/scheming/snippets/form_field.html

```diff
@@ -0,0 +1,23 @@
+{% ckan_extends %}
+
+{% block removal_popup%}
+  {% set locale = h.dump_json({'content': _('Are you sure you want to delete this status?') }) %}
```

Comment:
Need a translation for this string in the .po file

Reply:
By adding something like this to canada.po?
`#: ckanext/canada/templates/internal/scheming/snippets/form_field.html
msgid "Are you sure you want to delete this status?"
msgstr "Voulez-vous vraiment supprimer ce statut?"`

Reply:
yes like that, you can see an example of how @HoussamBedja did it in #1124